app/code/Magebit/Faq1/Model/ResourceModel/Question/CollectionFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magebit\Faq1\Model\ResourceModel\Question;
+
+use Magento\Framework\ObjectManagerInterface;
+
+/**
+ * I understand that this is service class that instantiate non-injectable class, a.k.a model that represent a database entity. It is a
+ * layer of abstraction between the ObjectManager and business code.
+ */
+class CollectionFactory
```

Comment:
This should be autogenerated, the Factory classes are autogenerated during code compilation, remove this file